commit 2cabb2588da56735369131b709f191453c080be0
Author: John W. Linville <linville@tuxdriver.com>
Date:   Fri Feb 14 13:58:44 2014 -0500

    crda: Add DESTDIR support in install-libreg* rules in Makefile
    
    Signed-off-by: John W. Linville <linville@tuxdriver.com>
    Signed-off-by: Luis R. Rodriguez <mcgrof@do-not-panic.com>

diff --git a/Makefile b/Makefile
index 0b2f0d7f8d4e..1f2550915c0d 100644
--- a/Makefile
+++ b/Makefile
@@ -120,13 +120,13 @@ $(LIBREG): regdb.h reglib.h reglib.c
 
 install-libreg-headers:
 	$(NQ) '  INSTALL  libreg-headers'
-	$(Q)mkdir -p $(INCLUDE_DIR)
-	$(Q)cp *.h $(INCLUDE_DIR)/
+	$(Q)mkdir -p $(DESTDIR)/$(INCLUDE_DIR)
+	$(Q)cp *.h $(DESTDIR)/$(INCLUDE_DIR)/
 
 install-libreg:
 	$(NQ) '  INSTALL  libreg'
-	$(Q)mkdir -p $(LIBDIR)
-	$(Q)cp $(LIBREG) $(LIBDIR)/
+	$(Q)mkdir -p $(DESTDIR)/$(LIBDIR)
+	$(Q)cp $(LIBREG) $(DESTDIR)/$(LIBDIR)/
 	$(Q)ldconfig
 
 %.o: %.c regdb.h $(LIBREG)
